src/components/Sidebar/Sidebar.tsx

```diff
@@ -46,15 +46,20 @@ const SidebarNav = styled.nav`
     }
   }
 `;
+const VercelStyle = styled.a`
+  display: block;
+  width: 10px;
+  color: #f90606;
```

Comment:
Is this color necessary?  If not, can it be removed?

Also, I think we could use some space above and below the banner.  Maybe we can do a swap?

```suggestion
  margin: 20px 0 10px;
```